core/text/formatters.py

```diff
@@ -181,29 +308,20 @@ def every_word(word_func):
     "kebab": "DASH_SEPARATED",
     "packed": "DOUBLE_COLON_SEPARATED",
     "padded": "SPACE_SURROUNDED_STRING",
-    "slasher": "ALL_SLASHES",
-    "conga": "SLASH_SEPARATED",
+    "lasher": "SLASH_SEPARATED",
     "smash": "NO_SPACES",
     "snake": "SNAKE_CASE",
     "string": "SINGLE_QUOTED_STRING",
 }
+
 prose_formatter_names = {
     "say": "NOOP",
     "speak": "NOOP",
-    "sentence": "CAPITALIZE_FIRST_WORD",
+    "sentence": "SENTENCE",
```

Comment:
`CAPITALIZE_FIRST_WORD` is in use in several other places and should be preserved even if it is a misnomer. (Sorry!)

Reply:
done